An exception occurred while executing event SessionLoginEvent

https://dump.geysermc.org/fFDmll6DtFPbew8KxQ1yJHXcaEQYj1Uj

`[11:01:37] [GeyserServerChild-4-1/WARN] [geyser]: Extension GeyserFloatingPoints. loads class it.unimi.dsi.fastutil.longs.Long2ObjectMap from an external source. This can change at any time and break the extension, additionally to potentially causing unexpected behaviour!
[11:01:37] [GeyserServerChild-4-1/ERROR] [org.geysermc.geyser.event.GeyserEventBus]: An exception occurred while executing event SessionLoginEvent for subscriber org.geysermc.geyser.event.GeyserEventSubscriber
java.lang.NoClassDefFoundError: it/unimi/dsi/fastutil/longs/Long2ObjectMap
        at org.oryxel.gfp.session.CachedSession.<init>(CachedSession.java:40) ~[?:?]
        at org.oryxel.gfp.geyser.GFPExtension.onSessionJoin(GFPExtension.java:27) ~[?:?]
        at org.lanternpowered.lmbda.Lmbda$27.accept(Unknown Source) ~[velocity-3.4.0-SNAPSHOT-521.jar:3.4.0-SNAPSHOT (git-81deb1ff-b521)]
        at org.geysermc.event.subscribe.impl.SubscriberImpl.lambda$new$0(SubscriberImpl.java:55) ~[?:?]
        at org.geysermc.event.subscribe.impl.SubscriberImpl.invoke(SubscriberImpl.java:75) ~[?:?]
        at org.geysermc.event.bus.impl.BaseBusImpl.fireSilently(BaseBusImpl.java:201) ~[?:?]
        at org.geysermc.event.bus.impl.OwnedEventBusImpl.fireSilently(OwnedEventBusImpl.java:41) ~[?:?]
        at org.geysermc.event.bus.impl.BaseBusImpl.fire(BaseBusImpl.java:180) ~[?:?]
        at org.geysermc.event.bus.impl.OwnedEventBusImpl.fire(OwnedEventBusImpl.java:41) ~[?:?]
        at org.geysermc.geyser.session.GeyserSession.connectDownstream(GeyserSession.java:1044) ~[?:?]
        at org.geysermc.event.bus.impl.OwnedEventBusImpl.fireSilently(OwnedEventBusImpl.java:41) ~[?:?]
        at org.geysermc.event.bus.impl.BaseBusImpl.fire(BaseBusImpl.java:180) ~[?:?]
        at org.geysermc.event.bus.impl.OwnedEventBusImpl.fire(OwnedEventBusImpl.java:41) ~[?:?]
        at org.geysermc.geyser.session.GeyserSession.connectDownstream(GeyserSession.java:1044) ~[?:?]
        at org.geysermc.geyser.session.GeyserSession.authenticate(GeyserSession.java:908) ~[?:?]
        at org.geysermc.geyser.network.UpstreamPacketHandler.handle(UpstreamPacketHandler.java:268) ~[?:?]
        at org.cloudburstmc.protocol.bedrock.packet.ResourcePackClientResponsePacket.handle(ResourcePackClientResponsePacket.java:20) ~[?:?]
        at org.cloudburstmc.protocol.bedrock.packet.BedrockPacketHandler.handlePacket(BedrockPacketHandler.java:9) ~[?:?]
        at org.cloudburstmc.protocol.bedrock.BedrockSession.onPacket(BedrockSession.java:119) ~[?:?]
        at org.cloudburstmc.protocol.bedrock.BedrockPeer.onBedrockPacket(BedrockPeer.java:69) ~[?:?]
        at org.cloudburstmc.protocol.bedrock.BedrockPeer.channelRead(BedrockPeer.java:258) ~[?:?]
        at io.netty.channel.AbstractChannelHandlerContext.fireChannelRead(AbstractChannelHandlerContext.java:359) ~[velocity-3.4.0-SNAPSHOT-521.jar:3.4.0-SNAPSHOT (git-81deb1ff-b521)]
        at io.netty.handler.codec.MessageToMessageDecoder.channelRead(MessageToMessageDecoder.java:107) ~[velocity-3.4.0-SNAPSHOT-521.jar:3.4.0-SNAPSHOT (git-81deb1ff-b521)]
        at io.netty.handler.codec.MessageToMessageCodec.channelRead(MessageToMessageCodec.java:120) ~[velocity-3.4.0-SNAPSHOT-521.jar:3.4.0-SNAPSHOT (git-81deb1ff-b521)]
        at io.netty.channel.AbstractChannelHandlerContext.fireChannelRead(AbstractChannelHandlerContext.java:357) ~[velocity-3.4.0-SNAPSHOT-521.jar:3.4.0-SNAPSHOT (git-81deb1ff-b521)]
        at io.netty.handler.codec.MessageToMessageDecoder.channelRead(MessageToMessageDecoder.java:107) ~[velocity-3.4.0-SNAPSHOT-521.jar:3.4.0-SNAPSHOT (git-81deb1ff-b521)]
        at io.netty.channel.AbstractChannelHandlerContext.fireChannelRead(AbstractChannelHandlerContext.java:359) ~[velocity-3.4.0-SNAPSHOT-521.jar:3.4.0-SNAPSHOT (git-81deb1ff-b521)]
        at io.netty.handler.codec.MessageToMessageDecoder.channelRead(MessageToMessageDecoder.java:107) ~[velocity-3.4.0-SNAPSHOT-521.jar:3.4.0-SNAPSHOT (git-81deb1ff-b521)]
        at io.netty.handler.codec.MessageToMessageCodec.channelRead(MessageToMessageCodec.java:120) ~[velocity-3.4.0-SNAPSHOT-521.jar:3.4.0-SNAPSHOT (git-81deb1ff-b521)]
        at io.netty.channel.AbstractChannelHandlerContext.fireChannelRead(AbstractChannelHandlerContext.java:357) ~[velocity-3.4.0-SNAPSHOT-521.jar:3.4.0-SNAPSHOT (git-81deb1ff-b521)]
        at io.netty.handler.codec.MessageToMessageDecoder.channelRead(MessageToMessageDecoder.java:107) ~[velocity-3.4.0-SNAPSHOT-521.jar:3.4.0-SNAPSHOT (git-81deb1ff-b521)]
        at io.netty.channel.AbstractChannelHandlerContext.fireChannelRead(AbstractChannelHandlerContext.java:359) ~[velocity-3.4.0-SNAPSHOT-521.jar:3.4.0-SNAPSHOT (git-81deb1ff-b521)]
        at io.netty.handler.codec.MessageToMessageDecoder.channelRead(MessageToMessageDecoder.java:107) ~[velocity-3.4.0-SNAPSHOT-521.jar:3.4.0-SNAPSHOT (git-81deb1ff-b521)]
        at io.netty.handler.codec.MessageToMessageCodec.channelRead(MessageToMessageCodec.java:120) ~[velocity-3.4.0-SNAPSHOT-521.jar:3.4.0-SNAPSHOT (git-81deb1ff-b521)]
        at io.netty.channel.AbstractChannelHandlerContext.fireChannelRead(AbstractChannelHandlerContext.java:357) ~[velocity-3.4.0-SNAPSHOT-521.jar:3.4.0-SNAPSHOT (git-81deb1ff-b521)]
        at io.netty.channel.DefaultChannelPipeline$HeadContext.channelRead(DefaultChannelPipeline.java:1429) ~[velocity-3.4.0-SNAPSHOT-521.jar:3.4.0-SNAPSHOT (git-81deb1ff-b521)]
        at io.netty.channel.DefaultChannelPipeline.fireChannelRead(DefaultChannelPipeline.java:918) ~[velocity-3.4.0-SNAPSHOT-521.jar:3.4.0-SNAPSHOT (git-81deb1ff-b521)]
        at org.cloudburstmc.netty.channel.raknet.RakChannelPipeline.lambda$onUnhandledInboundMessage$0(RakChannelPipeline.java:60) ~[?:?]
        at io.netty.util.concurrent.AbstractEventExecutor.runTask(AbstractEventExecutor.java:148) ~[velocity-3.4.0-SNAPSHOT-521.jar:3.4.0-SNAPSHOT (git-81deb1ff-b521)]
        at io.netty.util.concurrent.AbstractEventExecutor.safeExecute(AbstractEventExecutor.java:141) ~[velocity-3.4.0-SNAPSHOT-521.jar:3.4.0-SNAPSHOT (git-81deb1ff-b521)]
        at io.netty.util.concurrent.SingleThreadEventExecutor.runAllTasks(SingleThreadEventExecutor.java:507) ~[velocity-3.4.0-SNAPSHOT-521.jar:3.4.0-SNAPSHOT (git-81deb1ff-b521)]
        at io.netty.channel.SingleThreadIoEventLoop.run(SingleThreadIoEventLoop.java:180) ~[velocity-3.4.0-SNAPSHOT-521.jar:3.4.0-SNAPSHOT (git-81deb1ff-b521)]
        at io.netty.util.concurrent.SingleThreadEventExecutor$5.run(SingleThreadEventExecutor.java:1073) ~[velocity-3.4.0-SNAPSHOT-521.jar:3.4.0-SNAPSHOT (git-81deb1ff-b521)]
        at io.netty.util.internal.ThreadExecutorMap$2.run(ThreadExecutorMap.java:74) ~[velocity-3.4.0-SNAPSHOT-521.jar:3.4.0-SNAPSHOT (git-81deb1ff-b521)]
        at io.netty.util.concurrent.FastThreadLocalRunnable.run(FastThreadLocalRunnable.java:30) ~[velocity-3.4.0-SNAPSHOT-521.jar:3.4.0-SNAPSHOT (git-81deb1ff-b521)]
        at java.base/java.lang.Thread.run(Thread.java:1583) [?:?]
Caused by: java.lang.ClassNotFoundException: it.unimi.dsi.fastutil.longs.Long2ObjectMap
        at org.geysermc.geyser.extension.GeyserExtensionClassLoader.findClass(GeyserExtensionClassLoader.java:107) ~[?:?]
        at org.geysermc.geyser.extension.GeyserExtensionLoader.classByName(GeyserExtensionLoader.java:147) ~[?:?]
        at org.geysermc.geyser.extension.GeyserExtensionClassLoader.findClass(GeyserExtensionClassLoader.java:97) ~[?:?]
        at org.geysermc.geyser.extension.GeyserExtensionClassLoader.findClass(GeyserExtensionClassLoader.java:79) ~[?:?]
        at java.base/java.lang.ClassLoader.loadClass(ClassLoader.java:593) ~[?:?]
        at java.base/java.lang.ClassLoader.loadClass(ClassLoader.java:526) ~[?:?]
        ... 42 more`

--- a/build.gradle
+++ b/build.gradle
@@ -23,16 +23,18 @@ dependencies {
     compileOnly("org.geysermc.geyser:core:2.7.1-SNAPSHOT") {
         exclude group: "com.google.code.gson", module: "gson"
     }
-    compileOnly("it.unimi.dsi:fastutil:8.5.15")
+
+    implementation("it.unimi.dsi:fastutil:8.5.15")
+    implementation("com.fasterxml.jackson.dataformat:jackson-dataformat-yaml:2.15.2")
+
     compileOnly("org.projectlombok:lombok:1.18.36")
     annotationProcessor("org.projectlombok:lombok:1.18.36")
-
-    implementation("com.fasterxml.jackson.dataformat:jackson-dataformat-yaml:2.15.2")
 }
 
 shadowJar {
     relocate 'org.yaml.snakeyaml', 'org.oryxel.gfp.shaded.snakeyaml'
     relocate 'com.fasterxml', 'org.oryxel.gfp.shaded.fasterxml'
+    relocate 'it.unimi.dsi.fastutil', 'org.oryxel.gfp.shaded.fastutil'
 }
 
 def targetJavaVersion = 17

